.github/workflows/ci.yaml

```diff
@@ -142,3 +141,24 @@ jobs:
           --exclude rosetta-server-ethereum \
           --exclude rosetta-server-polkadot \
           --exclude rosetta-client
+
+    - name: Cleanup Docker
+      run: ./scripts/reset_docker.sh
+  
+  #Running arbitrum nitro-testnode
+  arbitrum-nitro-testnode-script:
+    needs: [rustfmt, test-client]
+    runs-on: self-hosted
+    steps:
+    - name: Checkout code
+      run: git clone -b release --recurse-submodules https://github.com/ManojJiSharma/nitro-testnode.git
+
+    - name: Install Docker Compose
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y docker-compose
```

Comment:
Odd... docker compose is already installed in the VM, why we need to install it again?

Reply:
removed